README.md

```diff
@@ -64,6 +64,23 @@ in order to run the API service
 uvicorn ols.app.main:app --reload --port 8080
 ```
 
+### Optionally run with podman
+There is an all-in-one image that has the document store included already.
+
+1. Follow steps above to create your config yaml and your API key file(s). 
+1. Place your config yaml and your API key file(s) in a known location (eg:
+`/path/to/config``)
+1. Place your api key file(s) in the same folder (eg: `/path/to/config``)
+1. Make sure your config yaml references the config folder for the path to your
+key file(s) (eg: `credentials_path: config/openai_api_key.txt``)
```

Comment:
are all these double backticks doing some magic i'm not familiar with?

Reply:
user error. good catch.